jekyll/_cci2/artifacts.md

```diff
@@ -289,6 +291,15 @@ Placeholder   | Meaning
 `:build_num`  | The number of the job (aka. build) for which you want to download artifacts.
 {: class="table table-striped"}
 
+## Artifact storage customization
+{: #artifacts-and-self-hosted-runner }
+
+When using self-hosted runners, there is a network and storage usage limit included in your plan. There are certain actions related to artifacts that will accrue network and storage usage. Once your usage exceeds your limit, charges will apply.
+
+Retaining an artifact for a long period of time will have storage cost implications, therefore, it is best to determine why you are retaining artifacts. The benefit of retaining an artifact is that you can use it to troubleshoot why a build is failing. Once the build passes, the artifact is likely not needed. Setting a low storage retention for artifacts is recommended if this suits your needs.
```

Comment:
```suggestion
Retaining an artifact for a long period of time will have storage cost implications, therefore, it is best to determine why you are retaining artifacts. One benefit of retaining an artifact might be so you can use it to troubleshoot why a build is failing. Once the build passes, the artifact is likely not needed. Setting a low storage retention for artifacts is recommended if this suits your needs.
```

Reply:
I _think_ there are many many use cases for artifacts so might be better to be vague here